pkg/controller/controller_serviceclass.go

```diff
@@ -33,30 +37,65 @@ func (c *controller) serviceClassAdd(obj interface{}) {
 	c.serviceClassQueue.Add(key)
 }
 
+func (c *controller) serviceClassUpdate(oldObj, newObj interface{}) {
+	c.serviceClassAdd(newObj)
+}
+
+func (c *controller) serviceClassDelete(obj interface{}) {
+	serviceClass, ok := obj.(*v1beta1.ClusterServiceClass)
+	if serviceClass == nil || !ok {
+		return
```

Comment:
Is that a normal situation to get an object of a wrong type? Shouldn't we log error?
With generics it wouldn't even be possible to pass a wrong type, so I guess it's not okay.